Sources/ProjLintKit/Models/FileViolation.swift

```diff
@@ -13,6 +13,9 @@ class FileViolation: Violation {
     }
 
     override func logViolation() {
-        print("\(type(of: rule).name) Violation – \(message)", level: level.printLevel, file: path, line: line)
+        let fileUrl = URL(fileURLWithPath: FileManager.default.currentDirectoryPath)
+            .appendingPathComponent(path)
+            .standardized
```

Comment:
The absolute + standardized path is needed because, if the file path contains `/./` or `/../`, Xcode will not show the error next to the offending line